app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -0,0 +1,66 @@
+@import "../tokens/colors";
+$transition: 0.2s cubic-bezier(0.95, 0.05, 0.795, 0.035);
+
+[class^=pb_checkbox_kit] {
+  .pb_checkbox_label {
+    display: block;
+    position: relative;
+    padding-left: 30px;
+    cursor: pointer;
+    font-size: $text_lt_default;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  .pb_checkbox_checkmark {
+    position: absolute;
+    height: 22px;
+    width: 22px;
+    border: solid $border_light 2px;
+    border-radius: 4px;
+    .pb_icon_kit{
+      opacity:0;
+      position: relative;
+      top:-2px;
+      left:1px;
+      color: $white;
+    }
+  }
+
+  &:hover input ~ .pb_checkbox_checkmark {
+    border-color: $primary_action;
+  }
+
+  input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+    display: none;
+    &:checked ~ .pb_checkbox_checkmark {
+      background-color: $primary_action;
+      border-color: $primary_action;
+    }
+    &:checked ~ .pb_checkbox_checkmark {
+      display: block;
+      .pb_icon_kit{
+        opacity:1;
```

Comment:
This should use opacity tokens = [Opacity Tokens](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/tokens/_opacity.scss)

Reply:
add a space between the colon